src/articles/FindArticles.js

```diff
@@ -23,13 +23,12 @@ export default function NewArticles({ api }) {
 
   useEffect(() => {
     setDisplayedExtensionPopup(LocalStorage.displayedExtensionPopup());
-    console.log("Running in chrome desktop: " + runningInChromeDesktop());
     console.log(
       "Localstorage displayed extension: " +
         LocalStorage.displayedExtensionPopup()
     );
 
-    if (runningInChromeDesktop()) {
+    if (runningInChromeDesktop() || runningInFirefoxDesktop()) {
```

Comment:
shouldn't we, at least theoretically test also for the other browsers that we support? i.e. Edge?  
or is Edge reporting as Chrome? Then maybe rename the function name?

Reply:
Well, yeah. We should probably make a "runningInEdge" etc for all browsers supported. So we can specifically use that browser name in the message. But we have not implemented this yet :)